src/d3a/models/strategy/load_hours.py

```diff
@@ -241,10 +222,6 @@ def event_bid_traded(self, *, market_id, bid_trade):
                 f"Energy requirement for load {self.owner.name} fell below zero " \
                 f"({self.energy_requirement_Wh[market.time_slot]})."
 
-        self.remove_bid_from_pending(bid_trade.offer.id, market)
```

Comment:
Sorry, this line was just added in the last commit and it is not in `super().event_bid_traded()`
Can you please explain?

Reply:
Ah yes, this is a bit misleading here. The bid deletion is handled by 'add_bid_to_bought' method of the BidEnabledStrategy class (the super() of the LoadHoursStrategy). This method has an optional parameter called 'remove_bid' which is true by default, so by default it deletes the bid from the pending bids (self._bids) and moves it to the bought bids (self._traded_bids). Therefore there is no need for having the bid removal twice here, that's why I deleted this line.